### PR TITLE
Added ovs.pp for fixing ovs log flooding issue

### PR DESCRIPTION
**Pull request by satroutr**
Fixes: Bug#1208979

### CODE REVIEW

**Comment by rickerc** [comment]
Sorry, closed accidentally

**Comment by ghost** [comment]
testing this today. in the meanwhile, please run it through puppet lint and make the appropriate changes (should just be aligning the equal signs and hashrockets)

apt-get install puppet-lint if you don't already have it

**Comment by satroutr** [comment]
Push the new patch

File: manifests/ovs.pp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#
+# This module will expose the functionality for users
+# to alter the log level for open vswitch
+#
+define coe::ovs(
+  $facility  = 'facility',
```
Comment (ghost) [inline manifests/ovs.pp]: Let's set the defaults for facility and log_level to either actual available options, eg file and err, or to nothing at all. Picking random values is not an approach we'd like to use in our code.

Comment (markvoelker) [inline manifests/ovs.pp]: I'd tend to agree with this.  I believe the default behavior prior to this patch is "syslog" and "warn", so those might make sensible defaults.

**Comment by ghost** [comment]
:+1:

**Comment by markvoelker** [comment]
Wouldn't the defaults prior to the introduction of this class be syslog and warn?  Those would make more sensible defaults, IMHO.  Using info could get really chatty, and using file would direct messages to somewhere users aren't likely expecting them.

**Comment by satroutr** [comment]
@markvoelker the default parameter are for facility we can use either console for that its log_level will be OFF if use syslog then the log_level will be ERR and is we use file then log_level will be INFO

root@control-server:/home/localadmin# ovs-appctl vlog/list
                 console    syslog    file
                 -------    ------    ------
autopath           OFF        ERR       INFO
backtrace          OFF        ERR       INFO
bond               OFF        ERR       INFO

**Comment by markvoelker** [comment]
@satroutr : Ok, makes sense then.

**Pull request merged**